.codeclimate.yml

```diff
@@ -0,0 +1,29 @@
+---
+engines:
+  bundler-audit:
+    enabled: true
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+      - javascript
+      - python
+      - php
```

Comment:
Being a bit picky, but do we need anything other than Ruby in here? Same for the rating paths below.